Cython/Compiler/Optimize.py

```diff
@@ -2580,26 +2588,53 @@ def _handle_simple_function_set(self, node, function, pos_args):
                 is_temp=node.is_temp,
                 py_name="set"))
 
-    PyFrozenSet_New_func_type = PyrexTypes.CFuncType(
-        Builtin.frozenset_type, [
-            PyrexTypes.CFuncTypeArg("it", PyrexTypes.py_object_type, None)
-        ])
-
+    # When calling frozenset(), we replace SimpleCallNode
```

Comment:
Feel free to improve the documentation of the class. This is not a good place to put such a comment.
```suggestion
```